### PR TITLE
APEXCORE-725 Add username/password config to jdbc operator test.

### PR DESCRIPTION
**Pull request by tweise**
R: @amberarrow

### CODE REVIEW

File: library/src/test/resources/JdbcProperties.xml [hunk 1]
```diff
@@ -177,8 +177,15 @@
     </value>
   </property>
 
+   <!-- following properties serve as example, they can also be omitted for the test -->
```
Comment (amberarrow) [inline library/src/test/resources/JdbcProperties.xml]: example => examples

**Comment by amberarrow** [comment]
Looks OK, other than the minor comment.

**Comment by tweise** [comment]
done

**Comment by tweise** [comment]
@amberarrow can you please LGTM or merge the PR.

**Comment by amberarrow** [comment]
Will do

**Comment by amberarrow** [comment]
Not sure why the title on this PR says APEXCORE-725 -- maybe from an earlier overwritten commit.

**Pull request merged**